Zeeguu Reader/Zeeguu Reader/FeedOverviewTableViewController.swift

```diff
@@ -44,6 +44,8 @@ class FeedOverviewTableViewController: ZGTableViewController, AddFeedTableViewCo
 	override func viewDidLoad() {
 		super.viewDidLoad()
 		
+		NSNotificationCenter.defaultCenter().addObserver(self, selector: #selector(FeedOverviewTableViewController.userDidLogin(_:)), name: UserLoggedInNotification, object: nil)
```

Comment:
I will refuse to code review these long lines of code - they don't fit in the github pull request diff view! so you learn a lesson ;)